cmake/ctest/drivers/atdm/mutrino/local-driver.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -l
+
+if [ "${SRUN_CTEST_TIME_LIMIT_MINUTES}" == "" ] ; then
+  SRUN_CTEST_TIME_LIMIT_MINUTES=180  # Default limit is 3 hours
+fi
+
+set -x
+
+export CTEST_DO_TEST=OFF
+
+#/usr/local/bin/salloc -N 1 -p standard -J $JOB_NAME \
+#  --time=${SRUN_CTEST_TIME_LIMIT_MINUTES} \
+  $WORKSPACE/Trilinos/cmake/ctest/drivers/atdm/ctest-s-driver.sh
```

Comment:
Does this mean that it is running tests on the login node and not the compute node?

Reply:
```
export CTEST_DO_TEST=OFF
```
is turning off testing so it will not be run on the login.  The salloc is commented out so it will build on the login.  How are we handling the machines that need to build on the logins and test on computes?

Reply:
@fryeguy52, this has been worked out for several weeks on 'serrano'. See:

* https://github.com/trilinos/Trilinos/blob/develop/cmake/ctest/drivers/atdm/README.md#split-ctest--s-drivers-and-submits-to-cdash

and the usage of this for 'serrano' at:

* https://github.com/trilinos/Trilinos/blob/develop/cmake/ctest/drivers/atdm/toss3/local-driver.sh

NOTE: I am experimenting with 'sbatch' and 'srun' right to try to address the failures on 'serrano' described in #2699.  But using salloc on 'serrano' seems to work fine for just the Panzer test and example suite.